aiscripts/mule.lib.evaluate_tradeoffers.xml

```diff
@@ -109,7 +115,7 @@
 					</do_elseif>
 
 					<!-- how much does the need want? -->
-					<get_ware_reservation object="$buyoffer.owner" type="buy" ware="$buyoffer.ware" result="$needReservations" />
+					<!--<get_ware_reservation object="$buyoffer.owner" type="buy" ware="$buyoffer.ware" result="$needReservations" />-->
```

Comment:
Why is this commented out?

Reply:
Result is not used => CPU time waste. Or maybe i don't know about a side effect of this call.

Reply:
Ah, right. I had problems in the past with Mules picking up more of a ware than the destination would buy, and then the trade would error out into the debug log and bad things would happen. I went through a couple of attempts trying to solve that, but I think in the end, the call to clamp_trade_amount right after this is probably what is taking care of that.